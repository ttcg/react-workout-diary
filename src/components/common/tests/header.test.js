import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '../header';

it('renders without crashing', () => {    

    const component = shallow(<Header />);

    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
});