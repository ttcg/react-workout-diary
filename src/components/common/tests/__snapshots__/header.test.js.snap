// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Navbar
  color="dark"
  dark={true}
  expand="md"
  sticky="top"
  tag="nav"
>
  <Link
    className="navbar-brand"
    to="/"
  >
    React Workout Diary
  </Link>
  <NavbarToggler
    className="mr-2"
    onClick={[Function]}
    tag="button"
    type="button"
  />
  <Collapse
    appear={false}
    enter={true}
    exit={true}
    in={false}
    isOpen={false}
    mountOnEnter={false}
    navbar={true}
    onEnter={[Function]}
    onEntered={[Function]}
    onEntering={[Function]}
    onExit={[Function]}
    onExited={[Function]}
    onExiting={[Function]}
    tag="div"
    timeout={350}
    unmountOnExit={false}
  >
    <Nav
      className="mr-auto"
      navbar={true}
      tag="ul"
      vertical={false}
    >
      <NavItem
        tag="li"
      >
        <NavLink
          activeClassName="active"
          className="nav-link"
          exact={true}
          to="/"
        >
          Home
        </NavLink>
      </NavItem>
      <NavItem
        tag="li"
      >
        <NavLink
          activeClassName="active"
          className="nav-link"
          exact={true}
          to="/workoutsapi"
        >
          Workouts (Api)
        </NavLink>
      </NavItem>
      <NavItem
        tag="li"
      >
        <NavLink
          activeClassName="active"
          className="nav-link"
          exact={true}
          to="/workoutsaga"
        >
          Workouts (Saga)
        </NavLink>
      </NavItem>
      <NavItem
        tag="li"
      >
        <NavLink
          activeClassName="active"
          className="nav-link"
          exact={true}
          to="/workouts"
        >
          Workouts (Redux)
        </NavLink>
      </NavItem>
      <NavItem
        tag="li"
      >
        <NavLink
          activeClassName="active"
          className="nav-link"
          exact={true}
          to="/workoutsreact"
        >
          Workouts (React)
        </NavLink>
      </NavItem>
      <NavItem
        tag="li"
      >
        <NavLink
          activeClassName="active"
          className="nav-link"
          exact={true}
          to="/workoutsreacthooks"
        >
          Workouts (ReactHooks)
        </NavLink>
      </NavItem>
      <NavItem
        tag="li"
      >
        <NavLink
          activeClassName="active"
          className="nav-link"
          exact={true}
          to="/about"
        >
          About
        </NavLink>
      </NavItem>
    </Nav>
  </Collapse>
</Navbar>
`;
